README.md

```diff
@@ -192,6 +192,34 @@ def get_special_floors_list(default_list):
   return default_list
 ```
 
+## Monsters
+`DowntimeDungeon_monsters` - gvar_id
+Configures the list of potential monsters
+
+The corresponding gvar if set should contain a function called `get_monster_list` which takes the argument `default_list` and returns a `list` to use on the server.
+
+The configure_monster() function takes the list of arguments shown below.

```

Comment:
```suggestion
The get_monster_list() function takes the list of arguments shown below.
```

Reply:
This one is meant to be like that, didn't feel like writing out name, token, pp, ... again if its a few lines below

Reply:
I could probably make it clearer tho

Reply:
mb looks good